scalableminds/webknossos
# Show owner of tracing if activeUser != owner

To do:
- [x] we need to ensure that the owner of the tracing is only exposed if the active user has the adequate permissions (citing flo: "only show the owner if the viewing user is allowed to manage them. This is mostly interesting for team-managers and admins anyway.")

### URL of deployed dev instance (used for testing):
- https://showowner.webknossos.xyz

### Steps to test:
- open an own tracing --> no owner should be shown
- open a dataset --> no owner should be shown
- open a foreign tracing --> owner should be displayed (if user is allowed to see it)

### Issues:
- fixes #3706 

------
